frontend/src/features/modules/ModulesTree.tsx

```diff
@@ -1,60 +1,134 @@
 import { Disclosure, DisclosureButton, DisclosurePanel } from '@headlessui/react'
-import { ChevronRightIcon } from '@heroicons/react/24/outline'
+import {
+  ArrowRightCircleIcon,
+  BellAlertIcon,
+  BellIcon,
+  BoltIcon,
+  BookOpenIcon,
+  ChevronRightIcon,
+  CircleStackIcon,
+  CodeBracketSquareIcon,
+  Cog6ToothIcon,
+  DocumentDuplicateIcon,
+  LockClosedIcon,
+  NumberedListIcon,
+  SquaresPlusIcon,
+} from '@heroicons/react/24/outline'
+import { TableCellsIcon } from '@heroicons/react/24/solid'
```

Comment:
nit: Should we use `24/outline` for this one as well to be consistent? Also we can ignore this for now if that's easier since we'll probably use different icons soon.

Reply:
nice catch, yeah the outline version blends in a little better